lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
The use or the existence of the table is behind a flag.  They're different.  Also, flag name?